docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Managing Your Fleet with Viam's Cloud API"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
```

Comment:
```suggestion
description: "Use the cloud app API with Viam's client SDKs to manage your robot fleet with code."
```
^ this conveys the why - we should try to avoid starting descriptions with interrogative words